src/schrodinger/sketcher/resources/schrodinger_livedesign.qss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Stylesheet for WASM build of the 2D Sketcher
```

Comment:
From main.cpp, it looks like this style sheet is getting applied to all builds, not just the WASM one.  Is that intentional?